Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3167,6 +3169,18 @@ def gen2021HiMix(fragment,howMuch):
                                     '--era' : 'Run2_2018_pp_on_AA',
                                     '--filetype':'DQM',
                                     '-n':100},hiDefaults2018_ppReco])
+                                    
```

Comment:
Steps 'HARVESTDRUN3' and  'HARVESTRUN3' were not in #39217: can you add a comment?

Reply:
well, it's needed because the PR that added them was not backported.